### ISSUE TITLE
ninôhtêpayin is not recognized as nôhtêpayiw, which doesn't get a paradigm

### ISSUE THREAD

**Issue by aarppe** [UAlbertaALTLab/cree-intelligent-dictionary#396]
For whatever reason, _ninôhtêpayin_ is not analyzed to have the lemma _nôhtêpayiw_:

![image](https://user-images.githubusercontent.com/15003407/80151370-29f8e000-8577-11ea-87f4-56ddf4f0266f.png)

... even though the FST recognizes the form:

```
hfst-lookup -q src/analyser-gt-desc.hfst
ninôhtêpayin
ninôhtêpayin	nôhtêpayiw+V+AI+Ind+1Sg	0.000000

nôhtêpayiw
nôhtêpayiw	nôhtêpayiw+V+AI+Ind+3Sg	0.000000
nôhtêpayiw	nôhtêpayiw+V+II+Ind+3Sg	0.000000
```
Might this be an artifact of the ambiguity concerning the lemma? (and thus related to #395)?

**Comment by Madoshakalaka** [comment]
I downloaded the [newst releases](https://github.com/UAlbertaALTLab/plains-cree-fsts/releases/tag/20200330) of `descriptive-analyzer.hfstol` and verified our app is indeed using the latest file.

It turns out the latest `descriptive-analyzer.hfstol` file does not recognize _nôhtêpayin_


```
$ md5sum CreeDictionary/res/fst/crk-descriptive-analyzer.hfstol
400d4a338fcd9fe928959594a8119c7b  CreeDictionary/res/fst/crk-descriptive-analyzer.hfstol
$ md5sum ~/Downloads/crk-descriptive-analyzer.hfstol 
400d4a338fcd9fe928959594a8119c7b  /home/matt/Downloads/crk-descriptive-analyzer.hfstol

$ hfst-optimized-lookup CreeDictionary/res/fst/crk-descriptive-analyzer.hfstol
nôhtêpayin
nôhtêpayin      nôhtêpayin      +?
```

**Comment by Madoshakalaka** [comment]
@eddieantonio @aarppe

**Comment by Madoshakalaka** [comment]
oooh, I think you meant  _ninôhtêpayin_ instead of _nôhtêpayin_?? is that a typo in the fist sentence?

**Comment by aarppe** [comment]
Yes, there was a typo, but the issue name has things right. The inflected form should be _ninôhtêpayin_.

**Comment by aarppe** [comment]
This seems to have been fixed with #419. The remainder is a duplicate in the source XML, which should become a separate issue #420.
